modules/vaos/config/routes.rb

```diff
@@ -8,7 +8,9 @@
     resources :appointment_requests, only: :index do
       resources :messages, only: :index
     end
-    resources :systems, only: :index
+    resources :systems, only: :index do
+      resources :facilities, only: :index, controller: 'direct_scheduling_facilities'
```

Comment:
Its minor, but seems that since you've got a specific "kind" of facility here in this controller (direct_scheduling) it feels like there are other kinds (indirect_scheduling? who knows). Mapping Facilities to DirectSchedulingFacilities seems like a problem unless there are no other kinds of facilities.

Basically really wondering why you just didn't do `direct_scheduling_facilities` as the resource name or rename the controller.

Reply:
I checked in case I missed something in the commit history that would help explain it, but those are pretty terse commit messages.

Reply:
Not having a custom controller name conflicts with the existing facilities controller, this routing has facilities belonging to systems which is the case. I do like the idea of renaming the route direct_scheduling_facilities 👍- If our devs (@saneshark @jbalboni @narin) agree with the change I’ll implement that.

Reply:
Yeah, that make sense. Especially because I think we're now going to have a more generic clinics endpoint in the future.